config_podinit.go

```diff
@@ -1,16 +1,13 @@
 package evergreen
 
 import (
-	"github.com/mongodb/anser/bsonutil"
 	"github.com/mongodb/grip"
 	"github.com/pkg/errors"
 	"go.mongodb.org/mongo-driver/bson"
 	"go.mongodb.org/mongo-driver/mongo"
 	"go.mongodb.org/mongo-driver/mongo/options"
 )
 
-var podInitConfigKey = bsonutil.MustHaveTag(Settings{}, "PodInit")
```

Comment:
I would put a [`//nolint: deadcode, unused`](https://github.com/evergreen-ci/evergreen/blob/168aae8a383eb8a76041e89597bace57d02902ac/config_db.go#L15) above this rather than remove it because we generally keep BSON tag keys even if we're not using them.